README.md

```diff
@@ -7,6 +7,11 @@
 
```

Comment:
- [ ] **Optional:** You do not need to specify `Project Name: VET CLINIC`. Just `VET CLINIC` would serve.